### PR TITLE
Fix bug with computing output shape with fused_reduce

### PR DESCRIPTION
**Pull request by pfultz2**

### CODE REVIEW

**Review by causten — APPROVED** [review]
I tested it and this works.  No comment on the code

File: src/fuse_reduce.cpp [hunk 1]
```diff
@@ -65,18 +65,10 @@ struct fused_reduce
...
            }))
             MIGRAPHX_THROW("Dimenstion does not match the submodule.");
-        const auto& s = inputs.at(0);
-        auto lens     = s.lens();
-        if(lens != sm->get_output_shapes().front().lens())
-        {
-            for(const auto& axis : axes)
-            {
-                lens[axis] = 1;
-            }
```
Comment (TedThemistokleous) [inline src/fuse_reduce.cpp:75]: Are we not concerned about this case anymore then?

**Review by TedThemistokleous — CHANGES_REQUESTED** [review]
Fix CI and just a question about the change in reduce.

File: src/fuse_reduce.cpp (hunk 1 above)
Comment (pfultz2) [reply src/fuse_reduce.cpp:75]: This was pointless because the output shape already computes the dimensions.

**Review by TedThemistokleous — APPROVED** [review]

**Comment by TedThemistokleous** [comment]
Fix CI

**Pull request merged**